### ISSUE TITLE
border color overrides validated control borders

### ISSUE THREAD

**Issue by tsmith-sdm** [twbs/bootstrap#25805]
Border color (ex: border-primary) which contains an !important directive overrides valid / invalid border colors on validated inputs.

For example if you add border-primary to the City, State or Zip fields on the following form and click Submit form the borders no longer turn red:

https://getbootstrap.com/docs/4.0/components/forms/#custom-styles

**Comment by rafalp** [comment]
This is on purpose - utility classes have !important in them to make them surefire when used:

https://github.com/twbs/bootstrap/issues/22815#issuecomment-308325578

Is there a reason preventing you from swapping `border-primary` for `border-warning` instead of adding `border-warning`? Its easy-peasy with today js's view libs.

**Comment by mdo** [comment]
Nope, utilities are meant to override when mixed with other components.
